aws-solutions/digital-evidence-archive-on-aws
# feat: introduce audit service and writer implementation

- target latest ES2022
- add audit stack which includes audit log group, cloudtrail trail with a trail log group and s3 bucket
- add audit-service and write functionality

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
